app/src/ereferrals/java/org/eyeseetea/malariacare/domain/usecase/CheckCredentialsWithOrgUnitUseCase.java

```diff
@@ -0,0 +1,32 @@
+package org.eyeseetea.malariacare.domain.usecase;
+
+import org.eyeseetea.malariacare.data.database.utils.PreferencesEReferral;
+import org.eyeseetea.malariacare.domain.entity.Credentials;
+
+public class CheckCredentialsWithOrgUnitUseCase implements UseCase {
```

Comment:
This really is not a domain use case, use case is only login but in erreferrals has two steps but this steps should be encapsulated in LoginUseCase